docs/docsite/rst/dev_guide/testing_integration.rst

```diff
@@ -70,13 +70,17 @@ outside of those test subdirectories.  They will also not reconfigure or bounce
 
    Use the ``--docker-no-pull`` option to avoid pulling the latest container image. This is required when using custom local images that are not available for download.
 
-Run as follows for all POSIX platform tests executed by our CI system::
+Run as follows for all POSIX platform tests executed by our CI system in a fedora32 docker container::
 
-    ansible-test integration --docker fedora29 -v shippable/
+    ansible-test integration shippable/ --docker fedora32 --docker-no-pull
```

Comment:
The `--docker-no-pull` option isn't needed here.

Reply:
I know, wanted to point it out one more time and reduce a number of pulling images when unnecessary.

Reply:
ok, removed